core/src/main/java/jenkins/util/xstream/SafeURLConverter.java

```diff
@@ -38,7 +38,7 @@
  * for {@link URLStreamHandler#equals(URL, URL)} or {@link URLStreamHandler#hashCode(URL)}. 
  * Required to protect against SECURITY-637
  * 
- * @since TODO
+ * @since 2.121.3
```

Comment:
This is probably wrong / needs to take into account LTS backports (or as in this case, restricted security fixes).

Reply:
Not sure what you mean @daniel-beck. I was also originally surprised by it too, but checking manually `2.121.3` seemed indeed like the first version where this got introduced. I.e. was *not* a backport AFAICT.

Or are you saying we should probably not ever put LTS version there, but only the first weekly where a change landed in?

Reply:
@batmat I checked the script and you're looking for the oldest tag. If something is introduced in a security update, the LTS is built and tagged an hour or two before the weekly.

Reply:
@batmat Would be a clear improvement over what we have, so +1 for that suggestion.

Even fancier: Obtain both LTS and non-LTS, compare LTS baseline to non-LTS, include both if baseline older than non-LTS. (Perhaps with added `TODO: Check whether @Restricted in LTS`).